src/components/common/Footer/DesktopFooter.js

```diff
@@ -14,27 +14,35 @@ const { Title } = Typography;
 function DesktopFooter() {
   return (
     <>
-      <Row
-        justify="space-between"
-        style={{ maxWidth: '1100px', margin: '0 auto' }}
-      >
+      <Row justify="space-between">
```

Comment:
Curious if there's a way to remove the inline styles?

Reply:
What's interesting here is that this is inline styling, yes, but also a prop we're passing down to the `Row` component - you can see this listed out in their docs at [this link](https://ant.design/components/grid/#:~:text=0-,justify,-Horizontal%20arrangement). 😄